src/components/checkboxes/template.njk

```diff
@@ -18,6 +18,9 @@
   {% endif %}
 {% endfor %}
 
+{#- fieldset is false by default -#}
```

Comment:
Non-blocking: I think we could remove these comments, or make them more descriptive what the logic is doing.

Reply:
I added this comment only because it was not obvious at first that the component would have no fieldset by default.